### DISCUSSION TITLE
DietPi-Software | Hyperion-NG

### DISCUSSION THREAD

**Discussion by eadmaster** [MichaIng/DietPi#5939] [Software Request] [answered]
https://github.com/hyperion-project/hyperion.ng

> Low CPU load makes it perfect for SoCs like Raspberry Pi

Comes with a builtin webserver running on http://localhost:8090 by default

**Comment by MichaIng** [comment]
An ambient light software. It controls an attached LED device based on current screen content. Can be installed via APT repository: https://github.com/hyperion-project/hyperion.ng/blob/master/Installation.md

**Comment by eadmaster** [comment, marked as answer]
I've installed the [`Hyperion-2.0.14-Linux-armv7l.deb` package here](https://github.com/hyperion-project/hyperion.ng/releases/tag/2.0.14) and it is working fine.

**Reply by MichaIng** [reply]
Ah nice packages are provided for all architectures we support. This makes it simple to implement.
